tests/regression_tests/trigger_statepoint_restart/test.py

```diff
@@ -0,0 +1,132 @@
+import glob
+import os
+
+import openmc
+import pytest
+
+from tests.testing_harness import PyAPITestHarness
+from tests.regression_tests import config
+
+
+@pytest.fixture
+def model():
+
+    # Materials
+    materials = openmc.Materials()
+    mat = openmc.Material()
+    mat.set_density('g/cm3', 4.5)
+    mat.add_nuclide('U235', 1.0)
+    materials.append(mat)
+
+    # Geometry
+    sph = openmc.Sphere(r=10.0, boundary_type='vacuum')
+    cell = openmc.Cell(fill=mat, region=-sph)
+    geometry = openmc.Geometry([cell])
+
+    # Settings
+    settings = openmc.Settings()
+    settings.run_mode = 'eigenvalue'
+    settings.batches = 10
+    settings.inactive = 5
+    settings.particles = 200
+    settings.keff_trigger = {'type': 'std_dev', 'threshold': 0.002}
+    settings.trigger_max_batches = 1000
+    settings.trigger_batch_interval = 1
+    settings.trigger_active = True
+    settings.verbosity = 1 # to test that this works even with no output
+ 
+    # Tallies
+    tallies = openmc.Tallies()
+    t = openmc.Tally()
+    t.scores = ['flux']
+    tallies.append(t)
+ 
+    # Plots (none)
+    plots = openmc.Plots()
+
+    # Put it all together
+    model = openmc.model.Model(materials=materials,
+                               geometry=geometry,
+                               settings=settings,
+                               tallies=tallies,
+                               plots=plots)
```

Comment:
If plots are empty, you don't need to specify it:
```suggestion
    # Put it all together
    model = openmc.model.Model(materials=materials,
                               geometry=geometry,
                               settings=settings,
                               tallies=tallies)
```